.gitignore

```diff
@@ -16,3 +16,4 @@ test/tmp
 test/version_tmp
 tmp
 vendor/
+.idea/
```

Comment:
Ignoring files created by your editor should be a job for your global gitignore

Reply:
I disagree. Our modulesync for modules handles all common IDE and text editor files. It makes it easier for our contributors and really doesn't harm anything on our end.